README.md

```diff
@@ -16,19 +16,29 @@ We’re working to update the [Scratch website](https://scratch.mit.edu) to use
 We’re currently building Scratch using [React](https://facebook.github.io/react/) and [SCSS](http://sass-lang.com/documentation/file.SASS_REFERENCE.html). Here are some resources to help you get acquainted with how we’re working on the Scratch codebase:
 
 * [Style Guide](https://github.com/LLK/scratch-www/wiki/Style-Guide)
-* [Testing Guide](https://github.com/LLK/scratch-www/wiki/Testing-Guide)
+* [Testing Guide](https://github.com/LLK/scratch-www/wiki/Testing-Guide-for-Bugfixes)
 * [Localization Guide](https://github.com/LLK/scratch-www/wiki/Localization-Guide)
 * [Map of the repository](https://github.com/LLK/scratch-www/wiki/Repo-Map)
 
 
 ### Before Getting Started
 * Make sure you have node (v4.2 or higher) and npm [installed](https://docs.npmjs.com/getting-started/installing-node)
+We use npm (Node Package Manager) to maintain and update packages required to build the site.  
+
+### Update Packages
+When making changes it's important to make sure that all of the node packages are up to date. Some changes in the code are needed because of package updates, so make sure you do this. You can update the packages by running this command:
 
-### To Build
 ```bash
 npm install
+```
+
+### To Build
+To build a version of the server on your local machine you can run this command:
+
+```bash
 npm run build
 ```
+This constructs new files on your machine.  If you want the files to only be in memory for testing purposes, consider skipping to the To Run section below.
```

Comment:
> If you want to run a server that rebuilds the files as you edit them, skip to the To Run section below.

Reply:
One issue is that you need to have built the `intl` directory for the server to run properly. So if you skip the build step, all the text is message ids. Probably the fix is to add a step to the Run section that says `make translations`.